### ISSUE TITLE
Example of stuff to show teams

### ISSUE THREAD

**Issue by AntonyBishop** [ministryofjustice/cloud-platform#1472]
Showing team how to deploy to the cloud platform is not enough anymore, our example should also cover : 

- Complete deployment flow ( CI/CD ), following best practices
- What a proper monitoring / alerting (including pingdom) setup look like
- Sensible naming convention
- Sensible branching strategy (that may be too much?) 
- Pod autoscaling in response to increased load

Some of the above is already a part of the examples/user-guide, but the point is to have a good example of complete end-to-end flow.

**Comment by digitalronin** [comment]
I think it makes more sense to park this ticket until after we have merged the gitops work (#1158).

The best outcome from this ticket is to create more 'stuff' by default, at the end of the `make namespace` process (e.g. a hello world application deployment, SSL certificate, alerts, and monitoring). Most, if not all, of that stuff belongs in the user's application source code repository, rather than in the cloud platform environments repository, so it will be easier to do this ticket after the gitops work (which also involves adding files to the user's code repo) has been merged.
